numpy/lib/_iotools.py

```diff
@@ -677,7 +682,22 @@ def _loose_call(self, value):
 
     def _strict_call(self, value):
         try:
-            return self.func(value)
+
+            # We check if we can convert the value using the current function
+            new_value = self.func(value)
+
+            # In addition to having to check whether func can convert the
+            # value, we also have to make sure that we don't get overflow 
+            # errors for integers.
+            if self.func is int:
+                try:
+                    np.array([value], dtype=self.type)
+                except OverflowError:
+                    raise ValueError("")
```

Comment:
You don't need the parenthesis and empty string, `raise ValueError` alone is valid Python.